src/main/resources/languages/en-US.json

```diff
@@ -330,7 +330,7 @@
     "weather": {
       "success": "Set weather ID to %s with climate type %s.",
       "status": "Current weather ID is %s with climate type %s.",
-      "description": "Changes weather ID and climate type. Weather IDs can be found in WeatherExcelConfigData.json.\nClimate types: sunny, cloudy, rain, thunderstorm, snow, mist"
+      "description": "Changes weather ID and climate type. Weather IDs can be found in .\Resources\ExcelBinOutput\WeatherExcelConfigData.json.\nClimate types: sunny, cloudy, rain, thunderstorm, snow, mist"
```

Comment:
This is not valid json, if you want to include backslashes you need to escape them (e.g. `.\\Resources\\ExcelBinOutput\\WeatherExcelConfigData.json`)
I'd advise just using POSIX-style `/` path separators instead, they work on Windows too.

Reply:
Ok, thank you very much. At the same time, I found the Polish document "pl-pl. json" in "\ SRC \main\ Resources \ Languages" is not complete, I have changed it